Dockerfile.flower

```diff
@@ -0,0 +1,14 @@
+FROM python:3
+
+ENV PYTHONUNBUFFERED 1
+
+# Dockerfile based on the official Celery
+# https://github.com/docker-library/celery
+
+# Prepare app folders & cmd
+RUN pip install flower
+
+CMD ["flower", \
+     "--broker=amqp://admin:admin@rabbitmq:5672/comps", \
+     "--broker_api=http://admin:admin@rabbitmq:15672/api/", \
```

Comment:
these are hardcoded here... see how we did it on v1.5, you don't need this, there's an image